WhaleSpotting/react-app/src/components/Navbar.tsx

```diff
@@ -38,25 +42,55 @@ export default function Navbar(): JSX.Element {
                         className={CheckCurrentPage("Reportsighting")}
                         onClick={() => HandleLinkClick("Reportsighting")}>Report Sighting</Link>
                 </div>
-                <div className="changing-nav-links">
+                <div className="changing-nav-links" >
```

Comment:
can we remove the whitespace at the end here and in your other tags, we don't need a space before the `>`

Reply:
done